cves/CVE-2010-4476.yml

```diff
@@ -116,9 +127,16 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
+  answer: |
+    While somewhat obscure, this was found by Rick Regan,a blog poster,
+    in that the number 2.2250738585072011E-308 would hang a PHP application
+    while it tried to process the number, around the week of January 6, 2011.
+    On February 3rd, 2011, Bryan Sullivan from the Adobe Secure Software
+    Development Team discovered that this same number would cause a hang in Java
+    applications as well. There is a very small range of numbers that can cause
+    this hang due to the parsing of the number.
+  date: 2011-02-03
+  automated: false
   contest:
```

Comment:
Based on your above description, "contest" should be false

Reply:
I believe contest is only filled in if there was one, as the field's contents is the contest's name. That's what i got out of the description, at least.